changelog.md

```diff
@@ -1,3 +1,28 @@
+# 2.0.0
+
+### New Features
+* `index.js` file allows for ES6 destructing
+
+### Breaking Changes
+* Change name of `OpenBCIBoard` to `Cyton` to follow new naming convention.
+    Simply change:
+    ```ecmascript 6
+    const OpenBCIBoard = require('openbci').OpenBCIBoard;
+    const ourBoard = new OpenBCIBoard();
+    ```
+    ```ecmascript 6
+    const Cyton = require('openbci').Cyton;
```

Comment:
I didn't know what it meant until I looked it up. It's shorter, which is nice, but less explicitly descriptive, which is less nice